index.html

```diff
@@ -173,112 +203,303 @@ <h2 class="title-lg mg-1-rt">Alert with actions</h2>
+          <button class="btn primary-cart-btn">
+            <i class="fas fa-shopping-cart"></i>&nbsp; Add to cart</button>
 
-        <button class="btn primary-cart-btn">
-          <i class="fas fa-save"></i>&nbsp; Save</button>
+          <button class="btn primary-cart-btn">
+            <i class="fas fa-save"></i>&nbsp; Save</button>
 
-        <button class="btn primary-cart-outlined-btn">
-          <i class="fas fa-trash"></i>&nbsp; Delete</button><br />
+          <button class="btn primary-cart-outlined-btn">
+            <i class="fas fa-trash"></i>&nbsp; Delete</button>
 
+        </div><br />
       </div>
 
       <iframe class="container-lg mg-1-tb" src="https://carbon.now.sh/embed/XZpSeoloVxWzTJgM9ymE"
-        style="width: 992px; height: 220px; border:0; transform: scale(1); overflow:hidden;"
+        style="width: 43rem; height: 230px; border:0; transform: scale(1); overflow:hidden;"
         sandbox="allow-scripts allow-same-origin">
-      </iframe>
+      </iframe><br />
 
       <h2 class="mg-1-tb">Floating :</h2>
-      <div class="floating">
+      <div class="cont-shadow component-container pd-1-all">
+        <div class="floating container-lg">
 
-        <button class="btn primary-btn floating-btn icon-sm">
-          <i class="fas fa-shopping-cart"></i></button>
+          <button class="btn primary-btn floating-btn icon-sm">
+            <i class="fas fa-shopping-cart"></i></button>
 
-        <button class="btn primary-btn floating-btn icon-sm">
-          <i class="fas fa-plus"></i></button>
+          <button class="btn primary-btn floating-btn icon-sm">
+            <i class="fas fa-plus"></i></button>
 
-        <button class="btn primary-btn floating-btn icon-sm">
-          <i class="fas fa-location-arrow"></i></button>
+          <button class="btn primary-btn floating-btn icon-sm">
+            <i class="fas fa-location-arrow"></i></button>
 
+        </div><br />
       </div>
 
       <iframe class="container-lg mg-1-tb" src="https://carbon.now.sh/embed/WuooPlDVDFb3Gqq02K0I"
-        style="width: 992px; height: 220px; border:0; transform: scale(1); overflow:hidden;"
+        style="width: 43rem; height: 230px; border:0; transform: scale(1); overflow:hidden;"
         sandbox="allow-scripts allow-same-origin">
-      </iframe>
+      </iframe><br />
 
     </div>
     <hr />
 
+    <!-- avatar -->
+
+    <div id="avatar" class="pd-1-lr pd-1-bot">
+      <h1 class="mg-point8-tb">Avatar</h1>
+      <p>
+        Avatars can be used to represent people or objects. It supports images or letters.
+      </p>
+
+      <!-- avatar as images -->
+      <h2 class="mg-1-tb">Image Avatar</h2>
+
+      <div class="cont-shadow component-container pd-1-all mg-1-bot">
+        <div class="avatar-round mg-1-tb mg-2-lr">
+          <div class="avatar avatar-lg"><img loading="lazy" src="components/images/avatar1.jpeg"
+              class="img-responsive img-round" alt="">
```

Comment:
1. The indentation could be improved. 
2.  img tag does not have closing tag /.
3. alt tag is empty